core/lib/types/src/api/mod.rs

```diff
@@ -733,3 +735,76 @@ pub struct Proof {
     pub address: Address,
     pub storage_proof: Vec<StorageProof>,
 }
+
+/// Collection of overridden accounts, useful for `eth_estimateGas`.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct StateOverride(pub HashMap<Address, OverrideAccount>);
+
+/// Account override for `eth_estimateGas`.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct OverrideAccount {
```

Comment:
probably a matter of taste, but I'd prefer to use `ApiOverrideAccount` here and jut `OverrideAccount` instead of validated.
and do we really need `StateOverride` and `ValidatedStateOverride` ?

Reply:
Fixed in aa58304548629592f268fa1a96f03a769fc8f3c2